wasmtime/_func.py

```diff
@@ -110,6 +136,52 @@ def __call__(self, store: Storelike, *params: IntoVal) -> Union[IntoVal, Sequenc
         else:
             return results
 
+    def _call_raw(self, store: Storelike, *params: IntoVal) -> Union[IntoVal, Sequence[IntoVal], None]:
+        """
+        internal implementation of calling a function that uses `wasmtime_func_call_unchecked`
+        """
+        if getattr(self, "_ty", None) is None:
```

Comment:
Instead of falling back to `getattr` can the mypy-listed-type of `self._ty` be `Optional`?